### PR TITLE
composer: init new day1coloso fastcomposer

### PR DESCRIPTION
**Pull request by dongwonnn**
## init day1coloso fastcomposer 

- init day1coloso용 fastcomposer  
- vue2용 crpyto md4 코드 설정

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@day1co/fastcomposer",
-  "version": "0.16.1",
+  "name": "@day1coloso/fastcomposer",
+  "version": "0.0.1",
   "author": "fastcampus",
```
Comment (eunji1117) [inline package.json:4]: `day1coloso`..? 🤔

**Review by eunji1117 — APPROVED** [review]
LGTM!

**Review by kevinGwon — APPROVED** [review]
LGTM!

이전 버전을 바라보고 있었군요!

**Review by mitrvlr — APPROVED** [review]
gogo!

**Review by sangseophwang — APPROVED** [review]
LGTM !

**Pull request merged**